numpy/ma/core.py

```diff
@@ -6959,7 +6959,7 @@ def where(condition, x=_NoValue, y=_NoValue):
         element from `x`, otherwise from `y`.
     x, y : array_like, optional
         Values from which to choose. `x` and `y` need to have the same shape
-        as condition, or be broadcast-able to that shape.
+        as condition, or be broadcastable to that shape.
```

Comment:
Actually #8599 is merged now anyway - so this should be updated to be the same as `np.where`